app/src/components/Interface.tsx

```diff
@@ -99,7 +99,7 @@ const Interface = ({
               ? `${orderingTacos ? "Ordering" : "Order"} ${tacosToOrder} taco${
                   tacosToOrder > 1 ? "s" : ""
                 }`
-              : "No tacos"}
+              : "Order " + tacosToOrder + " Tacos"}
```

Comment:
Why is this changed from `No tacos` to `Order some Tacos`? Is this not when there are no tacos left to buy?